README.md

```diff
@@ -246,6 +248,7 @@ Sompolinsky Y, Lewenberg Y, Zohar A. '16.
 - [BigchainDB: A Scalable Blockchain Database](https://www.bigchaindb.com/whitepaper/bigchaindb-whitepaper.pdf). McConaghy T, Marques R, Müller A, De Jonghe D, McConaghy T, McMullen G, Henderson R, Bellemare S, Granzotto A. '17
 - 🎓 [Commitcoin: Carbon dating commitments with bitcoin](https://eprint.iacr.org/2011/677.pdf). Clark J, Essex A. FC '12.
 - [OpenTimestamps: Securing Software Updates using the Bitcoin Blockchain Financial Cryptography and Data Security](https://github.com/opentimestamps/). Todd P, Halpin H. (FC Poster '17)
+- 🎓 [Decentralized trusted timestamping using the crypto currency bitcoin](https://arxiv.org/pdf/1502.04015.pdf). Gipp B., Meuschke N., Gernandt A. iConference 2015
```

Comment:
This doesn't seem very novel. Commitcoin did it a few years before them. Likewise, Peter Todd's OpenTimestamps was developed in 2012, and they don't even seem to mention it in their paper, which is essentially the same idea... I'm not sure we want to include this.

Reply:
Well I don't think we have only very novel works here. This was also published in a venue, so somebody interested in timestamping may wanna take a look for any extra stuff. If you don't have any objections directly on the paper itself (eg mistakes) I think we should add it.